### PR TITLE
Create Native\Laravel\Facades\Notification

### PR DESCRIPTION
**Pull request by Mombuyish**
According to doc: https://nativephp.com/docs/1/the-basics/notifications that it needs Facade for notification, but missing now.

### CODE REVIEW

File: src/Facades/Notification.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Native\Laravel\Facades;
+
+use Illuminate\Support\Facades\Facade;
+
```
Comment (milwad-dev) [inline src/Facades/Notification.php:6]: Add notification methods into docblock!

Comment (Mombuyish) [reply src/Facades/Notification.php:7]: updated

Comment (milwad-dev) [reply src/Facades/Notification.php:7]: Just `Object_` is a problem!
I fix it after this PR

**Review by simonhamp — COMMENTED** [review]
I like the idea of consistency here, but you don't really need a facade for this class as it has its own factory: the `static new()` method

**Comment by Mombuyish** [comment]
I fixed all of suggestions and import failed class.

**Comment by mpociot** [comment]
> I like the idea of consistency here, but you don't really need a facade for this class as it has its own factory: the `static new()` method

I wanted to get rid of all these static constructors as they hide the Client dependency. That's why this was also removed in most other classes - so I think a facade is better suited in this case 👍

**Comment by mpociot** [comment]
Thank you

**Pull request merged**